src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int LENGTH_OF_BY = 4;
+    private static final int LENGTH_OF_AT = 4;
+    private static final int LENGTH_OF_EVENT = 6;
+    private static final int LENGTH_OF_DEADLINE = 9;
+    private static final int LENGTH_OF_TODO = 5;
+
     public static void main(String[] args) {
```

Comment:
The implementation for this method is quite long. Perhaps it could be abstracted further? For example, lines 34 - 50 could be abstracted to a new method for adding a new task. A similar approach can be adopted for abstracting the remaining code.